src/Path.hpp

```diff
@@ -2,22 +2,25 @@
 #pragma once
 
 #include "fmtmacros.hpp"
+
 #include "third_party/nonstd/string_view.hpp"
 
 #include <regex>
 #include <string>
 
-
-#define COMP_OPERATOR(OP) bool operator OP (const std::string& rhs) const { return origin OP rhs;}
+#define COMP_OPERATOR(OP)                                                      \

```

Comment:
Das mit dem Makro war eh nur "Quick and Dirty". im produktiv Code müsste es vermutlich wieder raus.  

Hat aber jetzt nichts mit der Formatierung zu tun.